### ISSUE TITLE
nipap-cli package issue

### ISSUE THREAD

**Issue by Aztechnologyy** [SpriteLink/NIPAP#1241]
Good morning Gentlemen,

1st thing first, thank you for the great job. 

That beeing said, I am creating an ansible script to install nipap. I actually found this yesterday night the following error while doing a apt -y install nipap-cli

```
root@xxxxxxxxxx:~# apt -y install nipap-cli
Reading package lists... Done
Building dependency tree       
Reading state information... Done
nipap-cli is already the newest version (0.29.8-1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up nipap-cli (0.29.8-1) ...
cp: cannot create regular file '/etc/bash_completion.d/nipap': No such file or directory
dpkg: error processing package nipap-cli (--configure):
 installed nipap-cli package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 nipap-cli
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Might be a small brocken package problem. I thought it would be a good idea to report asap since I didn't find any post on it.

for information :

Linux deb10-xxxxxx 4.19.0-8-amd64 #1 SMP Debian 4.19.98-1 (2020-01-26) x86_64 GNU/Linux

Have a nice day.

**Comment by froztbyte** [comment]
This is the same as issue #1248 that I just created - see that issue for further detail.

**Comment by garberg** [comment]
This should be fixed in #1280.
